webgpu.h

```diff
@@ -1566,6 +1568,37 @@ typedef struct WGPUAdapterInfo {
     /*.deviceID=*/0 _wgpu_COMMA \
 })
 
+/**
+ * Enabled by the "subgroups" feature.
+ * Provides bounds on the sizes of subgroups supported by the adapter.
+ *
+ * Default values can be set using @ref WGPU_ADAPTER_PROPERTIES_SUBGROUPS_INIT as initializer.
+ */
+typedef struct WGPUAdapterPropertiesSubgroups {
+    WGPUChainedStructOut * nextInChain;
+    /**
+     * If the "subgroups" feature is supported, the minimum supported subgroup size for the adapter.
+     *
+     * Defaults to `0`.
```

Comment:
Not sure if this should be UNDEFINED

Reply:
Maybe `WGPU_LIMIT_U32_UNDEFINED` (i.e., `default: constant.limit_u32_undefined` in `webgpu.yml`?

Reply:
This is only an output (not an input) so there's no special handling of UNDEFINED, and it's not really a limit anyway. Since there's no real meaning to either value I think the INIT struct should just do zero-init.

(BTW I am going to change the wording of "Defaults to" because I was talking to someone yesterday and we realized that that sounds like the defaulting occurs _inside_ the API, but what we mean is the INIT value.)